src/App.tsx

```diff
@@ -1,61 +1,116 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { ChangeEvent, useState } from 'react';
+import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+
+import { TodoList } from './components/TodoList/TodoList';
+import { User } from './components/types/User';
+import { Todo } from './components/types/Todo';
+
+function getUser(userId: number): User | null {
+  const foundUser = usersFromServer.find((user) => user.id === userId);
+
+  return foundUser || null;
+}
+
+export const todos: Todo[] = todosFromServer.map((todo) => ({
+  ...todo,
+  user: getUser(todo.userId),
+}));
+
+export const App: React.FC = () => {
+  const [title, setTitle] = useState('');
+  const [selectedUser, setSelectedUser] = useState(0);
+  const [todosList, setTodosList] = useState<Todo[]>(todos);
```

Comment:
TS already knows about type if you put an initial value
```suggestion
  const [todosList, setTodosList] = useState(todos);
```